File: GerenciaMusic360/ClientApp/app/shared/models/project-budget.ts
import { ICategory } from './category';
import { ProjectBudgetDetail } from './project-budget-detail';
export interface ProjectBudget {
    id: number;
    projectId: number;
    categoryId: number;
    categoryName: string;
    budget: number;
    spent: number;
    notes: string;
    projectBudgetDetail: ProjectBudgetDetail[];
    category: ICategory;
    statusRecordId: number;
}

export interface ProjectBudgetExtend extends ProjectBudget {
    saldo: number;
}

File: GerenciaMusic360.Entities/ProjectBudget.cs
using System;
using System.Collections.Generic;

namespace GerenciaMusic360.Entities
{
    public partial class ProjectBudget
    {

        public ProjectBudget()
        {
            ProjectBudgetDetail = new HashSet<ProjectBudgetDetail>();
        }

        public int Id { get; set; }
        public int ProjectId { get; set; }
        public int CategoryId { get; set; }
        public decimal Budget { get; set; }
        public decimal Spent { get; set; }
        public string Notes { get; set; }
        public short StatusRecordId { get; set; }
        public DateTime Created { get; set; }
        public string Creator { get; set; }
        public DateTime? Modified { get; set; }
        public string Modifier { get; set; }
        public DateTime? Erased { get; set; }
        public string Eraser { get; set; }

        public Category Category { get; set; }
        public IEnumerable<ProjectBudgetDetail> ProjectBudgetDetail { get; set; }
    }
}
